github/repos_deployments_test.go

```diff
@@ -87,6 +108,27 @@ func TestRepositoriesService_ListDeploymentStatuses(t *testing.T) {
 	}
 }
 
+func TestRepositoriesService_GetDeploymentStatus(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/deployments/3/statuses/4", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"id":4}`)
+	})
+
+	deployment, _, err := client.Repositories.GetDeploymentStatus("o", "r", 3, 4)
+	if err != nil {
+		t.Errorf("Repositories.GetDeployment returned error: %v", err)
```

Comment:
Copy-paste error here, it should be "Repositories.GetDeploymentStatus returned".